pipeline/eval/eval.py

```diff
@@ -72,6 +74,81 @@
     WANDB_AVAILABLE = False
 
 
+def tokenize_nospace(sentence, tokenizer):
+    """
+    Tokenize a sentence and return a list of tokens that have no space
+    """
+
+    def iter_tok():
+        for t in tokenizer.tokenize(sentence):
+            if t == tokenizer.SPACE_TOKEN:
```

Comment:
Ideally the ability to tokenize without spaces should be a property of `IcuTokenizer`

Reply:
Up to you